prac_05/word_occurrences.py

```diff
@@ -0,0 +1,20 @@
+def main():
+    # sorts the words alphabetically, and prints each word with its count.
+    text = input("Text: ")
```

Comment:
The current implementation is case-sensitive, meaning that "Hello" and "hello" will be treated as different words. If you want to ignore case, you can convert all words to lowercase as optional. Like "text = input("Text: ").lower()".